js/change-list.js

```diff
@@ -247,7 +268,7 @@ class ChangeList {
   initEventsTrampoline(trampoline) {
     this.trampoline = trampoline;
     trampoline.mounted = true;
-    this.eventHandler = function(event) {
+    this.eventHandler = function (event) {
```

Comment:
```suggestion
    this.eventHandler = function(event) {
```

Ditto :)

Reply:
Sorry. My editor seems to be opinionated about JS formatting. I'll turn it off.